datadog/tests/resource_datadog_sensitive_data_scanner_rule_test.go

```diff
@@ -213,7 +229,7 @@ resource "datadog_sensitive_data_scanner_group" "sample_group" {
 
 resource "datadog_sensitive_data_scanner_group" "new_group" {
 	name = "another group"
-	is_enabled = true
+	is_enabled = false
```

Comment:
is it needed to change this?

Reply:
Yes it is needed

Reply:
There's a mechanism that disables a group if it gets empty. During this test, there's a rule recreation so the rule is deleted then readded, so the group gets disabled. Easier in tests to simply let it be disabled